src/components/Select/Select.tsx

```diff
@@ -45,6 +45,22 @@ export interface SelectProps {
      * If set to true, the user can type the value to search from the available options.
      */
     isSearchable?: boolean;
+    /**
+     * This is to specify if multiple options can be selected.
+     */
+    isMulti?: boolean;
+    /**
+     * This is to leave the select menu open upon selection.
+     */
+    closeMenuOnSelect?: boolean;
+    /**
+     * This is to overwrite defaulted styles
+     */
+    styles?: any;
```

Comment:
nit: I believe the type for this should be `Partial<CSSProperties>` (CSSProperties should be a react import). With that said, have you confirmed that this actually applies styles when provided? I know we have several layers of prop passing in the Select component so just making sure.

Reply:
I'll test with that type. Yeah, when passed a styled component it applied them. I didn't try scss though.